routes/api_v1_0.py

```diff
@@ -56,7 +95,27 @@ def datasets_query_v1_0():
         Response -- Response object containing list of available datasets w/ some metadata.
     """
 
-    return routes.routes_impl.query_datasets_impl(request.args)
+    data = []
+    if 'id' not in request.args:
```

Comment:
Nit pick: to ditch the `not` keyword could we flip the if-else check? I would argue it's more _pythonic_ 🙄